packages/apps/desktop-viewer-test/src/frontend/components/viewer/ModelViewer.tsx

```diff
@@ -44,16 +45,17 @@ export const ModelViewer = ({
     });
   }, []);
 
-  const onIModelAppInitialized = useMemo(() => {
-    console.log("Initialized!!");
+  const onIModelAppInitialized = useCallback(async () => {
+    await FrontendDevTools.initialize();
```

Comment:
For some reason, why i try to initialize this, IModelApp.localization is undefined, seems to be using the esm ver here, but the rest of the app uses cjs... not sure whats going on